### ISSUE TITLE
podman-remote: commit: not tagging

### ISSUE THREAD

**Issue by edsantiago** [containers/podman#7114]
In window 1:
```console
$ ./bin/podman system service --timeout=0
```
In window 2:
```console
$ ./bin/podman-remote run --name a alpine true
$ ./bin/podman-remote commit a b
f95ed969832cf49b012d33ffa3e4e020df892d3acde8fb9e6167c176ec4d25eb
$ ./bin/podman-remote  images
REPOSITORY                    TAG     IMAGE ID      CREATED             SIZE
<none>                        <none>  f95ed969832c  About a minute ago  5.85 MB    <---- missing tags
docker.io/library/alpine      latest  a24bb4013296  8 weeks ago         5.85 MB
```
Expected:
```
REPOSITORY                    TAG     IMAGE ID      CREATED        SIZE
localhost/b                   latest  feb57509edd1  4 seconds ago  5.85 MB
docker.io/library/alpine      latest  a24bb4013296  8 weeks ago    5.85 MB
```
master @ 288ebec6e, root & rootless

**Comment by edsantiago** [comment]
It's a string length problem??!? If I commit as `bb` (two characters), it works fine:
```console
$ ./bin/podman-remote commit a bb
72c80484d8679099965be00d4f0e108bb878946a02aa917520120d501fdb1992
$ ./bin/podman-remote images
REPOSITORY                    TAG     IMAGE ID      CREATED        SIZE
localhost/bb                  latest  72c80484d867  4 seconds ago  5.85 MB
```

**Comment by ashley-cui** [comment]
closed via #7158
